src/imitation/data/serialize.py

```diff
@@ -0,0 +1,172 @@
+"""Serialization utilities for trajectories."""
```

Comment:
I understand you just moved the code into a separate file? Or are there any other additional changes? Hard to see based on the diff github is showing

Reply:
Otherwise all seems OK

Reply:
I moved the serialization related functions to a separate file but also changed the save/load functions as they now use the huggingface backend. Sorry I also cant find a way to generate a decent diff for this.